proto/sifnode/margin/v1/types.proto

```diff
@@ -0,0 +1,38 @@
+syntax = "proto3";
+package sifnode.margin.v1;
+import "gogoproto/gogo.proto";
+
+option go_package = "github.com/Sifchain/sifnode/x/margin/types";
+
+message GenesisState {
+}
+
+message MTP {
```

Comment:
This varies slightly from the doc , is that intentional ? 
https://hackmd.io/M4y3-r1wTre6zQ4HX8R0Qg#tags-Ready-for-Review

Reply:
The latest spec is in the documentation-private repo.